i18n-command.php

```diff
@@ -22,3 +22,12 @@
 		}
 	}
 ) );
+
+WP_CLI::add_command( 'i18n po2json', '\WP_CLI\I18n\Po2JsonCommand', array(
```

Comment:
I think I'd go with `make-json` for now, but I'd change a few arguments to better fit that naming:
- Add a `--source=<extension(s)>` to define what type of extensions to use for extracting the strings. This might not make immediate sense for our initial use case, but "making JSON files from translations" would not necessarily be restricted to one source. The `--source` flag would default to `js` but could also be `js,php` or anything else, basically.
- Rename the `--keep-source-strings` flag to `--prune` and invert its logic. `--prune` would default to `true` but could be disabled through `--no-prune` or `--prune=false` then.

This would make the command a more universal JSON file generation. Thoughts?